### PR TITLE
adds dhcpv6 support via extra tag

### PR DESCRIPTION
**Pull request by ticevi**
adds new v6 tag to run dhcpv6 server in Docker
#20

### CODE REVIEW

**Comment by robinsmidsrod** [comment]
@ticevi I would suggest implementing this using an environment variable instead, so that the same entrypoint can run either IPv4 (default) or IPv6. Then you won't have so much duplication of code, following the DRY principle.

**Comment by ticevi** [comment]
@robinsmidsrod good point. I have adapted my merge request

**Comment by robinsmidsrod** [comment]
@ticevi Good direction! Can you change it even further, so that it uses -e DHCPD_PROTOCOL=6 (with value 4 being the default) to make the choice, and just have a single run line in entrypoint so that this environment variable generates a `-4` or `-6` argument to the final run command?

File: README.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ on.
 A simple `run` script is also included which makes it quick to iterate on a
 configuration until you're satisfied.
 
+DHCPv6
+------
+
+To use a DHCPv6-Server you have to pass `DHCPv6` as enviroment variable
```
Comment (robinsmidsrod) [inline README.md]: Please update this documentation.

**Pull request merged**